src/scss/constants.scss

```diff
@@ -150,6 +150,11 @@ $z-layer-live-schedule: 20;
 $z-layer-live-schedule-download: 25;
 $z-layer-sidebar: 15;
 $z-layer-emergency-content: 15;
+$z-layer-art-content: 5;
+$z-layer-jazz-header: 1;
+$z-layer-user-list-label: 1;
+$z-layer-embed-iframe: 1;
+$z-layer-external-message: 1;
```

Comment:
In order to avoid conflicts of the z layer, maybe it would be prudent to make these relative to a single base layer, kind of like in https://github.com/sparkletown/sparkle/pull/2266/commits/42161887a642debdb6a6e035864b65dfc03b921e

Reply:
fixed 65d922a